app/javascript/controllers/maps_controller.js

```diff
@@ -84,4 +88,60 @@ export default class extends Controller {
   getCurrentLocation() {
     console.log('FUNCIONA!');
   }
+
+  bussinessMarkers() {
+    const icons = {
+      food: {
+        icon: foodIcon,
+      },
+      products: {
+        icon: productsIcon,
+      },
+      services: {
+        icon: servicesIcon,
+      },
+    };
+    const store = [
+      {
+        position: new google.maps.LatLng(19.265129, -103.710844),
+        type: 'food',
+      },
+      {
+        position: new google.maps.LatLng(19.265129, -103.720844),
+        type: 'products',
+      },
+      {
+        position: new google.maps.LatLng(19.265129, -103.700844),
+        type: 'services',
+      },
+      {
+        position: new google.maps.LatLng(19.262129, -103.710844),
+        type: 'food',
+      },
+      {
+        position: new google.maps.LatLng(19.265129, -103.690844),
+        type: 'products',
+      },
+      {
+        position: new google.maps.LatLng(19.264129, -103.710844),
+        type: 'services',
+      },
+    ];
+    const contentString = '<p>La fonda de Doña Luisa</p><p>+Info -></p>';
```

Comment:
Is this for testing purposes? Instead of having the name of the restaurant should be a variable calling for db information

Reply:
It was indeed a code for testing. We want the information of the database to be shown in this part of JS, but we still haven't achieved it as we wish, so we leave it as a base code for another issue.